lib/app.rb

```diff
@@ -32,8 +32,8 @@ def worker_fetch(repo)
 
 
 
-class RubyDeps < Sinatra::Base
-  include RubyDepsHelpers
+class CuttingEdge < Sinatra::Base
```

Comment:
Same comment as above

Reply:
Done. The Sinatra App is now part of the module CuttingEdge, creating the namespace CuttingEdge::App.